platforms/core-configuration/model-core/src/integTest/groovy/org/gradle/api/provider/PropertyAssignmentIntegrationTest.groovy

```diff
@@ -111,29 +112,32 @@ class PropertyAssignmentIntegrationTest extends AbstractIntegrationSpec {
         groovyBuildFile(inputDeclaration, inputValue, operation)
 
         expect:
+        if (GradleContextualExecuter.configCache && expectedResultUnderCC != _) {
+            expectedResult = expectedResultUnderCC
+        }
         runAndAssert("myTask", expectedResult)
 
         where:
```

Comment:
Added a new column named `expectedResultUnderCC` , which may be specified for cases where the actual value changes between CC and no-CC

Reply:
> Added a new column named `expectedResultUnderCC` , which may be specified for cases where the actual value changes between CC and no-CC

:thinking: given that we only need to tweak a couple of expected results, maybe conditionally defining `expectedResult` would be a tad better:
```
result(withoutCC: '[provider(?)]', withCC: '[fixed(class MyObject, a)]')
```

Reply:
Made the expected result conditional with an inline conditional expression